src/style.css

```
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.loupe-large {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  padding: 25px 0 26px 26px;
+}
+
+.loupe-small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  padding: 15px 0 16px 13px;
+}
+
+.searchBar {
+  height: 70px;
+  width: 100%;
+
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+
+  padding: 25px 0 23px 62px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.smallSearch {
+  height: 42px;
+
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 19px;
+
+  padding: 12px 0 10px 33px;
+}
+
+.searchBar::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+
+  position: relative;
+}
+
+.searchBar:hover {
+  font-family: "Avenir", sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.searchBar:focus {
+  font-family: "Avenir", sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+
+  font-size: 900;
+}
```

no need to repeat `font-family` for eac state and pseudoelement
```suggestion
.searchBar::placeholder {
  font-weight: 300;
  color: #3d4e61;

  position: relative;
}

.searchBar:hover {
  background: #fff;
  border: 1px solid #e1e7ed;
  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
  border-radius: 4px;
}

.searchBar:focus {
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  border: 1px solid #e1e7ed;
  border-radius: 3px;

  font-size: 900;
}
```